### PR TITLE
continue button not found bug fix - specific-access-wa-ui-test

### PR DESCRIPTION
**Pull request by deepthidoppalapudihmcts**
**Before creating a pull request make sure that:**

- [x ] commit messages are meaningful and follow good commit message guidelines
- [x ] README and other documentation has been updated / added (if needed)
- [ x] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

Bug was identified that continue button found while running specific access wa ui test.  Added a wait befor continue button which fixed this issue.

**Does this PR introduce a breaking change?** (check one with "x")

```
[x ] Yes
[ ] No
```

### CODE REVIEW

File: e2e/tests/api_wa_tests/1v1_unspec_ga_wa_specific_access_test.js [hunk 1]
```diff
@@ -3,10 +3,10 @@ const mpScenario = 'ONE_V_ONE';
 
 let civilCaseReference, gaCaseReference;
 
-Feature(' GA - WA Specific Access @api-wa');
+Feature(' GA - WA Specific Access @api-ga-wa');
```
Comment (automatemm) [inline e2e/tests/api_wa_tests/1v1_unspec_ga_wa_specific_access_test.js:6]: @deepthidoppalapudihmcts it is supposed to be @api-wa . please revert these changes

**Review by vasudevganesanhmcts — APPROVED** [review]

**Review by automatemm — DISMISSED** [review]
